testing/trino-product-tests/src/main/java/io/trino/tests/cli/TestTrinoLdapCli.java

```diff
@@ -260,6 +265,17 @@ public void shouldFailForUserWithColon()
         skipAfterTestWithContext();
     }
 
+    @Test(groups = {LDAP_AND_FILE_CLI, PROFILE_SPECIFIC_TESTS}, timeOut = TIMEOUT)
+    public void shouldRunQueryWithFile()
+            throws IOException
```

Comment:
throws Exception

Reply:
We already spoke about that. There is no convention to use general exceptions in test.

Reply:
Did we? Do you have some reference?

`throws` in tests are very uninteresting. Here you say that you don't care about exception, so saying explicitly about which exception you don't care is not useful. Also it is place that could potentially change if something in test body changes. Having `throws Exception` is working always. Code is written once.

Reply:
Correct me if I am wrong:
-throwing a specific exception is not a mistake of any sort.
-throwing a generic exception is commonly considered as an anti-pattern.
-Trino convetions state that test code needs to be as good as production code.
-There is an unspoken rule that test methods **may** throw a generic Exception.
-There is plenty of tests that throw specific exceptions in Trino, like all the other tests in this class.

There are people that dislike checked exceptions (many of them) but throwing Exception everywhere is not a solution and definitely should not be forced.

Reply:
>  throwing Exception everywhere

Please do not go to the extreme. 

I am talking about exception in tests only. Throwing `Exception` is ok as long no-one really wants to catch it. I mean to use `Exception` as unchecked exception.

> -throwing a generic exception is commonly considered as an anti-pattern.

That is true for API. Not for tests where testng needs to catch `Throwable` anyway as assertions throws `Error` not `Exception.

Up to you.